src/mono/sample/wasm/browser-bench/frame-main.js

```diff
@@ -0,0 +1,66 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+"use strict";
+class FrameApp {
+    async init({ BINDING }) {
+        const reachManagedReached = BINDING.bind_static_method("[Wasm.Browser.Bench.Sample] Sample.AppStartTask/ReachManaged:Reached");
+        await reachManagedReached();
+    }
+
+    reached() {
+        window.parent.resolveAppStartEvent("reached");
+    }
+}
+
+globalThis.frameApp = new FrameApp();
+
+let mute = false;
+createDotnetRuntime(({ BINDING }) => {
+    return {
+        disableDotnet6Compatibility: true,
+        configSrc: "./mono-config.json",
+        printErr: function () {
+            if (!mute) {
+                console.error(...arguments);
+            }
+        },
+        onConfigLoaded: () => {
+            window.parent.resolveAppStartEvent("onConfigLoaded");
+            // Module.config.diagnostic_tracing = true;
+        },
+        onDotnetReady: async () => {
+            window.parent.resolveAppStartEvent("onDotnetReady");
+            try {
+                await frameApp.init({ BINDING });
+                window.parent.resolveAppStartEvent("done");
```

Comment:
```suggestion
```
I don't see use of this event and it is causing

    TypeError: Cannot read properties of null (reading 'resolveAppStartEvent')
        at Object.onDotnetReady (frame-main.js:36)

as the iframe is probably already unparented at this point.